exercises/collatz-conjecture/example.js

```diff
@@ -0,0 +1,14 @@
+export default (n) => {
+  if (n <= 0) {
+    throw new Error('Only positive numbers are allowed');
+  }
+
+  const iterate = (number, step) => {
```

Comment:
Thanks @rchavarria 
I've fixed variable names (now they are `iterable`, `number` and `step`). I think `step` is a good option - readme specifically refers that the result of exercise should be the number of "steps", but I make it singular, so it won't be confused with array. I've also left the parameter for outer function named `n`, again, for consistency with readme.